Python/Product/TestAdapter.Executor/Pytest/PytestTestDiscoverer.cs

```diff
@@ -59,16 +59,16 @@ ITestCaseDiscoverySink discoverySink
 
             _logger = logger ?? throw new ArgumentNullException(nameof(logger));
 
-            var workspaceText = settings.IsWorkspace ? Strings.WorkspaceText : Strings.ProjectText;
-            LogInfo(Strings.PythonTestDiscovererStartedMessage.FormatUI(PythonConstants.PytestText, settings.ProjectName, workspaceText, settings.DiscoveryWaitTimeInSeconds));
+            //var workspaceText = settings.IsWorkspace ? Strings.WorkspaceText : Strings.ProjectText;
+            //LogInfo(Strings.PythonTestDiscovererStartedMessage.FormatUI(PythonConstants.PytestText, settings.ProjectName, workspaceText, settings.DiscoveryWaitTimeInSeconds));
 
             var env = InitializeEnvironment(sources, settings);
             var outputFilePath = Path.GetTempFileName();
             var arguments = GetArguments(sources, settings, outputFilePath);
```

Comment:
dont we want to change 'GetArguments' so that it only looks for tests in a single source file now?

Reply:
@bschnurr We might be able to do that for pytest so that it only discovers 1 test at a time, but for unittest the argument has to be a directory, can't be a path to a single file.